### ISSUE TITLE
Rustic-doc-setup fails

### ISSUE THREAD

**Issue by hhamud** [emacs-rustic/rustic#66]
I try to run the command `rustic-doc-setup` but this fails with this error message
```
Contacting host: raw.githubusercontent.com:443
Wrote /Users/user/.local/bin/rustic-doc-convert.sh
Wrote /Users/user/.local/bin/rustic-doc-filter.lua
Your version of fd is too old, please install a recent version, maybe through cargo.
Your version of pandoc is too old, please install a more recent version. See their github for more info.
Setup is converting std. If you want to convert local dependencies, activate rustic-doc-mode when you are in a rust project and run ‘rustic-doc-convert-current-package
```

even though I have the necessary versions of both pandoc and fd-find

```shell
user@192 ~ % pandoc --version
pandoc 3.5
Features: +server +lua
Scripting engine: Lua 5.4
User data directory: /Users/user/.local/share/pandoc
Copyright (C) 2006-2024 John MacFarlane. Web: https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
user@192 ~ % fd --version
fd 10.2.0
user@192 ~ % 
```

**Comment by psibi** [comment]
Does this PR fix it for you: https://github.com/emacs-rustic/rustic/pull/48 ?

**Comment by hhamud** [comment]
> Does this PR fix it for you: #48 ?

yes it does now.

**Comment by psibi** [comment]
Thanks for the confirmation, closing the issue since that PR has been merged now.
